frontend/src/app/spot/styles/tokens/media-queries.yml

```diff
@@ -8,6 +8,12 @@ props:
   spot-mq-action-bar-change:
     value: "'(min-width: 768px)'"
 
+  spot-mq-mobile:
+    value: "'(max-width: 679px)'"
+
+  spot-mq-desktop:
```

Comment:
Are these okayed by product?

Reply:
those are already used throughout the code, but I will double check with @marcalcobe and @psatyal.

Reply:
Hi @b12f and @Kharonus,

I suppose this comes with the fact that the modal in mobile scales its width depending on the resolution of the phone. And that probably 680px is the smallest tablet we still target as mobile. In any case is weird that is 679px and I can't think of any other reason why is implemented like this in other places.

Reply:
okayed by product